docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+
+Een verzoek kan meerdere producten/diensten omvatten hetgeen een winkelmandje mogelijk maakt op de website van de gemeente. Als gevolg hiervan kan een verzoek tot meerdere zaken leiden. 
+
+# Gegevens: generiek en specifiek
+Bij een verzoek wordt alleen generieke metadata opgeslagen. Procesgegevens (domeinspecifiek) worden in een andere registratie opgeslagen. Bijv. bij een Melding Openbare Ruimte worden specifieke gegevens zoals locatie en meldinggegevens in een meldingen-registratie opgeslagen. Vanuit het verzoek wordt verwezen met een URI naar de melding.
+
+Wat betreft behandeltermijn (uiterste en geplande) volgen we de zaaktypen. Terzijde: deze zouden eigenlijk bij de producten in de PDC moeten staan, omdat producten de interface vormen tussen klant en dienstverlener. Voor nu volgen we het huidige model.
+
+# Typen verzoeken
+Er zijn verschillende typen verzoeken. Minimaal de volgende typen worden onderscheiden:
+* aanvragen product of dienst (uit de PDC van de gemeente)
+* doen van een melding (bijv. een melding openbare ruimte)
+* aanleveren gegevens bij een reeds bestaand verzoek
+* bezwaar maken (in feite geen verzoek om een product), maar wel een verzoek
+
+# Relaties/cardinaliteiten
+* Een contactmoment kan over een verzoek of een zaak gaan. Met een contactmoment kan geen verzoek worden ingediend. Meerdere contactmomenten kunnen aan een verzoek worden gekoppeld. 1:n van verzoek naar contacten.
+* De relatie tussen zaken en verzoeken is m:n. Een verzoek kan tot meerdere zaken leiden en een zaak kan gekoppeld zijn aan meerdere verzoeken (zie eerder voorbeeld over MOR).
+* Een verzoek kan meerdere producten omvatten. 1:n van verzoek naar producten
+* Een verzoek kan betrekking hebben op meerdere andere verzoeken. Bijv. het verzoek tot intrekken heeft betrekking op een of meer andere verzoeken. Het verzoek met aanvullende gegevens heeft betrekking op een ander verzoek. 1:n van verzoek naar verzoeken.
+* Een verzoek kan bijlagen hebben in de vorm van informatieobjecten. 1:n
+
+# Relatie met DSO-verzoeken
+Het moet mogelijk zijn om DSO-verzoeken te registreren in de Verzoeken API; er moet dus een mapping zijn van de attributen van een DSO-verzoek naar een ZGW Verzoek. Ook de relaties tussen DSO-Verzoeken moeten kunnen worden geregistreerd; zie ook de sectie over relaties hiervoor. De procesgegevens (ook wel domeinspecifieke gegevens) kunnen niet in de Verzoeken API worden opgeslagen en moeten in een vergunningen-registratie worden opgeslagen.
```

Comment:
> er moet dus een mapping zijn van de attributen van een DSO-verzoek naar een ZGW Verzoek
Je schrijft hier specifiek "ZGW Verzoek", we bouwen een generieke verzoeken API toch? dan is het geen ZGW verzoek neem ik aan?